Ginger/GingerCoreTest/Misc/DataSourceTest.cs

```diff
@@ -0,0 +1,264 @@
+#region License
+/*
+Copyright © 2014-2019 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+using Amdocs.Ginger.Common;
+using GingerCore.DataSource;
+using GingerCoreNET.DataSource;
+using GingerTestHelper;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Data;
+using GingerCore;
+using Ginger.Run;
+using GingerCore.Actions;
+using Amdocs.Ginger.Repository;
+
+namespace UnitTests.NonUITests
+{
+    [TestClass]
+    public class DataSourceTest
+    {
+        static AccessDataSource accessDataSource = new GingerCore.DataSource.AccessDataSource();
+        ObservableList<DataSourceTable> sourceTables;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field 'sourceTables' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4855725)